### PR TITLE
Sadie/test hotfix

### PR DESCRIPTION
**Pull request by sadief**
`destroy buyTokens` is still required. Without it we get the follow error - I believe because it is not being handled?
```error: cannot deploy invalid contract
 --> 9140375397c156bdf5e5f6215c6f574beb1908dfcf11a25249cc91f3c7ef02cf:4:2
 |
4 | signer.contracts.add(name: name, code: code.decodeHex())
 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
error: loss of resource
 --> 120e725050340cab.TopShotMarketV3:197:42
 |
197 | pub fun purchase(tokenID: UInt64, buyTokens: @FungibleToken.Vault): @TopShot.NFT {

### CODE REVIEW

**Review by rrrkren — APPROVED** [review]

**Comment by sadief** [comment]
@EricLin2004 I may need your help to override the code owner approval check since Josh is on vacation now until the 25th

**Review by EricLin2004 — APPROVED** [review]

**Pull request merged**
